x-pack/test/functional/apps/managed_content/managed_content.ts

```diff
@@ -112,15 +112,17 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       it('maps', async () => {
         await PageObjects.common.navigateToActualUrl(
           'maps',
-          'map/managed-d7ab-46eb-a807-8fed28ed8566'
+          'map/managed-d7ab-46eb-a807-8fed28ed8566',
+          { ensureCurrentUrl: false }
```

Comment:
Can you explain a bit why it sure ignore this check?

Reply:
Sometimes the redirect doesn't happen on time (redirect from `map/managed-d7ab-46eb-a807-8fed28ed8566` to `#map/managed-d7ab-46eb-a807-8fed28ed8566`) so the test is flaky, but the further lines of this test would test if the dashboard updates so I think we're good with ignoring it. 
It was also there before, but I removed it and it made the test flaky (here's Drew's explanation because we used this code before in some tests, but I removed it when attempting to fix flakiness the first time):  
```
// for some reason the URL didn't always match the expected, so I turned off this check
// URL doesn't matter as long as we get the dashboard app)
```